test/StringUtility.test.cpp

```diff
@@ -39,4 +48,8 @@ TEST(String, stringFrom) {
 	EXPECT_THAT(StringFrom(0.0f), testing::StartsWith("0.0"));
 	EXPECT_THAT(StringFrom(0.0), testing::StartsWith("0.0"));
 	EXPECT_THAT(StringFrom(0.0l), testing::StartsWith("0.0"));
+
+	// Implicit string conversion
+	ImplicitStringConversionTestFixture implicitStringConversionTestFixture{ "testString" };
+	EXPECT_EQ("testString", StringFrom(implicitStringConversionTestFixture));
```

Comment:
This is testing the test code. Perhaps it should be separated from the `stringFrom` test code.

Or is this really testing the compiler itself? Should we be testing this? Maybe we should assume the compiler does the correct thing and call the `std::string` conversion operator.

Reply:
I would defer to your thoughts if it should be moved or altogether removed. I was trying to test that implicit string conversion worked without throwing an error, etc.

Reply:
Wait a minute, after re-reading this with fresh eyes it's making more sense. Yes, we do want to test that.

I think my tired eyes had deleted the `StringFrom` call. Without it, it would just be testing that the test object can be automatically converted to a string in an equality comparison. With that, it does actually test the `StringFrom` project code, and makes sure it can handle types with auto conversion operators.

We should keep it.